agent_setup.sh

```diff
@@ -1019,6 +1025,12 @@ function main(){
     log_func "${FUNCNAME[0]}"
     initialize
 
+    if [[ "${IS_CONTAINER_AGENT}" == "true" ]]; then
+        echo "Container Agent - Agent setup script"
+        main_request_configuration_from_ims
+        exit 0
+    fi
+
     log "Starting the agent setup script"
```

Comment:
This log should go before the check for the container-agent var.

Reply:
moved & squashed.